packages/universal-components/src/Button/ButtonInner.js

```diff
@@ -20,8 +21,26 @@ type Props = {|
   +sublabel?: React.Node,
   +label?: React.Node,
   +circled?: boolean,
+  +size?: ButtonSize,
 |};
 
+const paddingTokens = {
+  paddingButton: 'paddingButton',
+  paddingButtonWithIcons: 'paddingButtonWithIcons',
+  paddingButtonWithLeftIcon: 'paddingButtonWithLeftIcon',
+  paddingButtonWithRightIcon: 'paddingButtonWithRightIcon',
+};
+
+const getPadding = (leftIcon, rightIcon) => {
```

Comment:
It's not returning actual padding so something like `getPaddingToken` or  `getPaddingType` will be probably more suitable.